test/Test/Pos/Ssc/GodTossing/ComputeSharesSpec.hs

```diff
@@ -29,6 +32,19 @@ spec = describe "computeSharesDistr" $ do
     prop totalStakeZeroDesc totalStakeIsZero
     prop validRichmenStakesWorksDesc validRichmenStakesWorks
     prop lrcConsistencyDesc lrcConsistency
+
+    describe "distributionNotBig" $ do
```

Comment:
I think it makes more sense to use regular English words here instead of something that reminds identifiers in code because these are not identifiers.